keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -205,7 +215,9 @@ data class ResourceActuationLaunched(
   override val application: String,
   val plugin: String,
   val tasks: List<Task>,
-  override val timestamp: Instant
+  override val timestamp: Instant,
+  override val level: EventLevel = EventLevel.INFO,
+  override val displayName: String = "Updating resource state",
```

Comment:
"updating resource state" is a bit confusing for me -  it doesn't convey that we started a task. But I don't have a better suggestion, so this comment is not that actionable lol